synapse/handlers/federation.py

```diff
@@ -311,8 +311,8 @@ async def _maybe_backfill_inner(
 
             forward_event_ids = await self.store.get_successor_events([bp.event_id])
 
-            extremities_events = await self.store.get_events(
-                forward_event_ids,
+            extremities_events = await self.store.get_events_as_list(
+                successor_event_ids,
```

Comment:
I don't think this name is in scope here, as of this commit (https://github.com/matrix-org/synapse/commit/cb880815a9cebafa37c3ef1aca0a17bdae6c03c7)

Reply:
ugh, sorry. It should be `forward_event_ids`. This gets fixed up later in the patch set.